src/code42cli/cmds/cases.py

```diff
@@ -261,3 +264,60 @@ def remove(state, case_number, event_id):
         state.sdk.cases.file_events.delete(case_number, event_id)
     except Py42NotFoundError:
         raise Code42CLIError("Invalid case-number or event-id.")
+
+
+@file_events.group(cls=OrderedGroup)
+@sdk_options(hidden=True)
+def bulk(state):
+    """Tools for executing file events actions associated to a case in bulk."""
+    pass
+
+
+FILE_EVENTS_HEADERS = [
+    "case_number",
```

Comment:
I think these need to be the same keys that occurs in py42 responses.
This way, it is easier to pipe the results of output into CSV input.

So,
`number` and `eventId`